l10n/de.js

```diff
@@ -1,6 +1,7 @@
 OC.L10N.register(
   "sociallogin",
   {
+    "Log in with": "Anmelden mit",
```

Comment:
It is useless here. This sentence is not used in js

Reply:
Ok :-) I'll remove it. Thought this might be useful for  the future.

Reply:
Was removed :-)